### PR TITLE
Add .github for CODEOWNERS

### PR DESCRIPTION
**Pull request by seemethere**
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#61598 Add .github for CODEOWNERS**

I'd like to be notified on changes to the github actions workflows, add
this so I can be notified.

Signed-off-by: Eli Uriegas <eliuriegas@fb.com>

Differential Revision: [D29685783](https://our.internmc.facebook.com/intern/diff/D29685783)

### CODE REVIEW

**Comment by seemethere** [comment]
@seemethere has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D29685783).

**Review by samestep — APPROVED** [review]

File: CODEOWNERS [hunk 1]
```diff
@@ -38,3 +38,8 @@
 
 # Docker
 /.circleci/docker/ @jeffdaily @jithunnair-amd
+
+# Github Actions
+# This list is for people wanting to be notified every time there's a change
+# related to Github Actions
+/.github/ @seemethere
```
Comment (walterddr) [inline CODEOWNERS:45]: should we add others from the team?

**Review by walterddr — APPROVED** [review]

File: CODEOWNERS (hunk 1 above)
Comment (seemethere) [reply CODEOWNERS:45]: Yup, If you'd like to be included on this list please comment below, I'm planning on merging this on Thursday July 15th to give other people time to comment here

Comment (janeyx99) [reply CODEOWNERS:45]: me

**Review by janeyx99 — APPROVED** [review]

File: CODEOWNERS (hunk 1 above)
Comment (zhouzhuojie) [reply CODEOWNERS:45]: me pls

**Review by malfet — APPROVED** [review]

File: CODEOWNERS (hunk 1 above)
Comment (driazati) [reply CODEOWNERS:45]: i nominate myself

**Comment by seemethere** [comment]
@seemethere has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D29685783).

**Pull request closed without merging**